pkg/bench/rttanalysis/orm_queries_bench_test.go

```diff
@@ -161,6 +161,25 @@ WHERE
 			Stmt:  `SELECT * FROM pg_attribute`,
 		},
 
+		{
+			Name: "has_database_privilege",
+			// prevents database exists error
+			Setup: `DROP DATABASE IF EXISTS d; CREATE DATABASE d`,
+			Stmt:  `SELECT has_database_privilege('d', 'CREATE')`,
+		},
```

Comment:
can you add more tests to see how it scales with number of databases/schemas/sequences?

basically, in the setup, it can create multiple entities, and the Stmt to test can be something like
```
SELECT has_database_privilege(name, 'CREATE') FROM crdb_internal.databases;
```

for schemas, something like
```
SELECT has_schema_privilege(nspame, 'CREATE') FROM pg_catalog.pg_namespace
```

Reply:
Talked about the tests not supporting has_database_privilege offline. Updated the others.